src/components/StateEligibility.js

```diff
@@ -118,3 +172,67 @@ export default function StateEligibility() {
         </div>
     );
 }
+
+function CriterionGroup({ className, group }) {
+    return (
+        <div className={className}>
+            <MarkupLink
+                text={group.title}
+                link={group.link}
+                variant="subtitle2"
+            />
+        </div>
+    );
+}
+
+function CriterionItem({ index, criterion, classes }) {
+    const now = new Date();
+
+    // skip any criteria that haven't started yet.
+    if (criterion.startDate && now < new Date(criterion.startDate)) {
+        return false;
+    }
+
+    return (
+        <ListItem key={"item" + index} className={classes.listItem}>
+            <ListItemIcon className={classes.listItemIcon}>
+                <PeopleIcon color={criterion.color} />
+            </ListItemIcon>
+            <ListItemText>
+                <MarkupLink link={criterion.link} text={criterion.text} />
+            </ListItemText>
+        </ListItem>
+    );
+}
+
+/*
+ * @param {string} text
+ * @param {string} link
+ * @param {string} variant
+ */
+function MarkupLink({ text, link, variant }) {
```

Comment:
This is a clever way to simplify the info we need to provide when creating the links!

Reply:
I didn't verify the regex is 100% correct but I trust it from the output visual result